packages/core/src/external/cda/remove-b64.ts

```diff
@@ -1,45 +1,53 @@
 import { toArray } from "@metriport/shared";
-import { XMLBuilder } from "fast-xml-parser";
 import { createXMLParser } from "@metriport/shared/common/xml-parser";
+import { XMLBuilder } from "fast-xml-parser";
 import { BINARY_MIME_TYPES } from "../../util/mime";
 
 const notesTemplateId = "2.16.840.1.113883.10.20.22.2.65";
 const resultsTemplateId = "2.16.840.1.113883.10.20.22.2.3.1";
 const b64Representation = "B64";
 
-export function removeBase64PdfEntries(payloadRaw: string): string {
+export function removeBase64PdfEntries(payloadRaw: string): {
+  documentContents: string;
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  b64Attachments: any[]; // TODO: improve typing
+} {
   const parser = createXMLParser({
     ignoreAttributes: false,
-    attributeNamePrefix: "@_",
+    attributeNamePrefix: "_",
     removeNSPrefix: true,
   });
   const json = parser.parse(payloadRaw);
 
   let removedEntry = 0;
 
+  //eslint-disable-next-line @typescript-eslint/no-explicit-any
+  const b64Attachments: any[] = [];
+
   if (json.ClinicalDocument?.component?.structuredBody?.component) {
     //eslint-disable-next-line @typescript-eslint/no-explicit-any
     toArray(json.ClinicalDocument.component.structuredBody.component).forEach((comp: any) => {
       if (
         toArray(comp.section?.templateId).some(
           //eslint-disable-next-line @typescript-eslint/no-explicit-any
           (template: any) =>
-            template?.["@_root"] === notesTemplateId || template?.["@_root"] === resultsTemplateId
+            template?.["_root"] === notesTemplateId || template?.["_root"] === resultsTemplateId
```

Comment:
why the change here? were we not flagging stuff before?

Reply:
So all the CDA types that we have defined in the codebase come without the `@`. So to reuse those types, I'm removing this character from being added by the XML parser, and therefore from all the fields that used it

Reply:
Is this the only place that needs the change... seems like a good idea but want to make sure we're not relying on `@` anywhere else

Reply:
I'll do the due diligence to make sure it doesn't cause troubles downstream